src/app/dim-ui/Sheet.tsx

```diff
@@ -53,6 +57,12 @@ export default function Sheet({
   const sheetContents = useRef<HTMLDivElement | null>(null);
   const sheetContentsRefFn = useLockSheetContents(sheetContents);
 
+  useEffect(() => {
```

Comment:
Kinda just a random question, would useLayoutEffect or useEffect be more optimal here?

Reply:
Yup I settled on `useEffect`